### PR TITLE
add tmp change to README

### PR DESCRIPTION
**Pull request by couchds**
Demonstrating a PR from a feature branch into main.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # cheerbot
 This repository will contain a Discord bot for the Garden of Cheer, a Discord Server dedicated to wholesome and healthy communication.
+
+Temporary change to show how branching works in git.
```
Comment (thesophiatan) [inline README.md:4]: Temporary text declined for main

Comment (couchds) [reply README.md:4]: why?

Comment (thesophiatan) [reply README.md:4]: I don't want temporary text in the read me because I want it to be as evergreen a document as possible.

Comment (couchds) [reply README.md:4]: fine i'll close the PR then

**Pull request closed without merging**